### PR TITLE
treewide: Basic cleanup and alignment to package guidelines

### PR DESCRIPTION
**Pull request by diizzyy**
This is a very basic cleanup, several packages needs more work
but this at least drops git for https and removes the use of
PKG_MD5SUM and some minor fixes.

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>

### CODE REVIEW

**Comment by diizzyy** [comment]
@tohojo
@simonwonderlich
@axn
@lynxis
@fingon
@mwarning
@bittof
@acinonyx 
@ecsv

**Comment by ecsv** [comment]
There is no package changed which is maintained by @simonwunderlich or me. vis and batmand are handled by @elektra42

But the URL change looks fine to me.

**Comment by tohojo** [comment]
LGTM

**Comment by acinonyx** [comment]
Looks good

**Comment by aparcar** [comment]
I think I can speak for bmx/axn and also approve

File: quagga/Makefile [hunk 1]
```diff
@@ -10,10 +10,10 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=quagga
 PKG_VERSION:=1.1.0
 PKG_RELEASE:=1
-PKG_MD5SUM:=daa303871e07ea5856aae6fd79e89722
+PKG_HASH:=f7a43a9c59bfd3722002210530b2553c8d5cc05bfea5acd56d4f102b9f55dc63
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://download.savannah.gnu.org/releases/quagga/
+PKG_SOURCE_URL:=@SAVANNAH/quagga/
```
Comment (aparcar) [inline quagga/Makefile]: doing this you could use also `@GITHUB` at other places

Comment (diizzyy) [reply quagga/Makefile]: No, that will break things...
https://github.com/lede-project/source/blob/master/scripts/download.pl#L205

**Comment by bittorf** [comment]
i'am fine with the changed in olsrd.

**Comment by mwarning** [comment]
meh, didn't mean to squash it.

**Pull request merged**